components/RecipeForm.js

```diff
@@ -45,98 +46,137 @@ export default function RecipeForm({ onSubmit }) {
     onSubmit(newData);
   }
   return (
-    <form onSubmit={handleSubmit}>
-      <StyledTop>
+    <Fragment>
```

Comment:
you only need <Fragment> instead of <> if you need to assign a key prop.
So you could save yourself the import and some letters and go with this:
```suggestion
    <>
```